### ISSUE TITLE
Cannot start any project in 3.21.0-beta.1

### ISSUE THREAD

**Issue by eldair** [lando/core#121]
Hello,
I have updated lando to the latest version today and am unable to start any project - here is the error message

`Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/run/desktop/mnt/host/wsl/docker-desktop-bind-mounts/Ubuntu/fbca942db8f70dc7d5150f9e042d5580d249ce84d490da08534929fbb15e1870" to rootfs at "/opt/bitnami/nginx/conf/lando.conf": mount /run/desktop/mnt/host/wsl/docker-desktop-bind-mounts/Ubuntu/fbca942db8f70dc7d5150f9e042d5580d249ce84d490da08534929fbb15e1870:/opt/bitnami/nginx/conf/lando.conf (via /proc/self/fd/6), flags: 0x5000: no such file or directory: unknown`

Lando is installed in wsl (2) not in windows directly, docker version is 4.27.2.
If I run lando destroy and then lando start, it works - but if I run lando start on a stopped project it throws the error above.  

Also, if I restart docker and start container via docker ui it will start but if I try lando start (which fails) then it becomes impossible to start it again via docker ui - restarting docker will again allows me to start containers via docker ui.

**Comment by pirog** [comment]
@eldair can you share your landofile so we can try to replicate?

**Comment by eldair** [comment]
@pirog sure, it happens in any app, here are two lando files

```
name: master_db
services:
  database:
    type: mysql:8.0
    portforward: 3600
    creds:
      user: test
      password: test
    run:
      - mysql -uroot < /app/script.sql
      - gunzip < /app/dump.sql.gz | mysql -uroot
```
this is `script.sql` content (it doesn't affect anything but if you need it).
```
GRANT ALL PRIVILEGES ON *.* TO 'test'@'%';
FLUSH PRIVILEGES;
```
second lando file:
```
name: dibz
recipe: laravel
proxy:
    appserver_nginx:
        - dibz.test
config:
    php: '8.2'
    via: nginx
    cache: redis
    database: mysql:8.0
    xdebug: false
    webroot: public
services:
    database: disabled
    appserver_nginx:
        scanner:
            okCodes:
                - 401
```

**Comment by eldair** [comment]
@pirog Is this issue fixed in latest update(s)? PHP 8.3 is unusable without v3.21

**Comment by eldair** [comment]
@pirog It seems to be fixed now, I will close the issue
